#: ../duplicity/manifest.py:249
msgid ""
msgstr ""
"Project-Id-Version: duplicity\n"
"Report-Msgid-Bugs-To: Kenneth Loafman <kenneth@loafman.com>\n"
"POT-Creation-Date: 2025-06-19 10:05-0500\n"
"PO-Revision-Date: 2025-06-19 15:16\n"
"Last-Translator: \n"
"Language-Team: French\n"
"Language: fr_FR\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n > 1);\n"
"X-Crowdin-Project: duplicity\n"
"X-Crowdin-Project-ID: 539508\n"
"X-Crowdin-Language: fr\n"
"X-Crowdin-File: /main/po/duplicity.pot\n"
"X-Crowdin-File-ID: 6\n"

#: ../duplicity/__main__.py:76
msgid "INT intercepted...exiting."
msgstr "INT intercepté... sortie"

#: ../duplicity/__main__.py:84
#, python-format
msgid "GPG error detail: %s"
msgstr "Détails de l’erreur GPG : %s"

#: ../duplicity/__main__.py:91
#, python-format
msgid "User error detail: %s"
msgstr "Détails de l’erreur utilisateur : %s"

#: ../duplicity/__main__.py:98
#, python-format
msgid "Backend error detail: %s"
msgstr "Détails de l’erreur de sauvegarde : %s"

#: ../duplicity/argparse311.py:291
msgid "usage: "
msgstr "utilisation: "

#: ../duplicity/argparse311.py:761
#, python-format
msgid "argument %(argument_name)s: %(message)s"
msgstr "argument %(argument_name)s: %(message)s"

#: ../duplicity/argparse311.py:870
msgid ".__call__() not defined"
msgstr ".__call__() non définie"

#: ../duplicity/argparse311.py:1115
#, python-format
msgid "conflicting subparser: %s"
msgstr "sous-analyseur en conflit : %s"

#: ../duplicity/argparse311.py:1118
#, python-format
msgid "conflicting subparser alias: %s"
msgstr "conflicting subparser alias: %s"

#: ../duplicity/argparse311.py:1152
#, python-format
msgid "unknown parser %(parser_name)r (choices: %(choices)s)"
msgstr "analyseur %(parser_name)r inconnu (choix : %(choices)s)"

#: ../duplicity/argparse311.py:1215
#, python-format
msgid "argument \"-\" with mode %r"
msgstr "argument \"-\" avec le mode %r"

#: ../duplicity/argparse311.py:1223
#, python-format
msgid "can't open '%(filename)s': %(error)s"
msgstr "ne peut pas ouvrir «%(filename)s» : %(error)s"

#: ../duplicity/argparse311.py:1427
#, python-format
msgid "cannot merge actions - two groups are named %r"
msgstr "ne peut pas fusionner les actions - deux groupes sont nommés %r"

#: ../duplicity/argparse311.py:1462
msgid "'required' is an invalid argument for positionals"
msgstr "'requis' est un argument invalide pour les positions"

#: ../duplicity/argparse311.py:1483
#, python-format
msgid "invalid option string %(option)r: must start with a character %(prefix_chars)r"
msgstr "chaîne d'option %(option)r invalide : doit commencer par un caractère %(prefix_chars)r"

#: ../duplicity/argparse311.py:1500
#, python-format
msgid "dest= is required for options like %r"
msgstr "dest= est requis pour les options comme %r"

#: ../duplicity/argparse311.py:1517
#, python-format
msgid "invalid conflict_resolution value: %r"
msgstr "valeur de résolution de conflit invalide : %r"

#: ../duplicity/argparse311.py:1534
#, python-format
msgid "conflicting option string: %s"
msgid_plural "conflicting option strings: %s"
msgstr[0] "chaîne d'options en conflit : %s"
msgstr[1] "chaînes d'options en conflit : %s"

#: ../duplicity/argparse311.py:1595
msgid "mutually exclusive arguments must be optional"
msgstr "les arguments mutuellement exclusifs doivent être facultatifs"

#: ../duplicity/argparse311.py:1670
msgid "positional arguments"
msgstr "arguments positionnels"

#. Used in usage help. (Should be consistent with the "Options:"
#. header.) Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/argparse311.py:1671 ../duplicity/cli_data.py:364
#: ../duplicity/cli_data.py:535 ../duplicity/cli_data.py:589
#: ../duplicity/cli_data.py:724 ../duplicity/cli_data.py:924
msgid "options"
msgstr "Options"

#: ../duplicity/argparse311.py:1689
msgid "show this help message and exit"
msgstr "afficher ce message d'aide et quitter"

#: ../duplicity/argparse311.py:1721
msgid "cannot have multiple subparser arguments"
msgstr "ne peut pas avoir plusieurs arguments de sous-analyseur"

#: ../duplicity/argparse311.py:1769 ../duplicity/argparse311.py:2264
#, python-format
msgid "unrecognized arguments: %s"
msgstr "Arguments non reconnus : %s"

#: ../duplicity/argparse311.py:1868
#, python-format
msgid "not allowed with argument %s"
msgstr "non autorisé avec l'argument %s"

#: ../duplicity/argparse311.py:1912 ../duplicity/argparse311.py:1926
#, python-format
msgid "ignored explicit argument %r"
msgstr ""

#: ../duplicity/argparse311.py:2030
#, python-format
msgid "the following arguments are required: %s"
msgstr "les arguments suivants sont requis : %s"

#: ../duplicity/argparse311.py:2042
#, python-format
msgid "one of the arguments %s is required"
msgstr "un des arguments %s est requis"

#: ../duplicity/argparse311.py:2083
msgid "expected one argument"
msgstr "attendait un argument"

#: ../duplicity/argparse311.py:2084
msgid "expected at most one argument"
msgstr "attendu au plus un argument"

#: ../duplicity/argparse311.py:2085
msgid "expected at least one argument"
msgstr "attendu au moins un argument"

#: ../duplicity/argparse311.py:2089
#, python-format
msgid "expected %s argument"
msgid_plural "expected %s arguments"
msgstr[0] "Attribut %s attendu"
msgstr[1] "arguments %s attendus"

#: ../duplicity/argparse311.py:2143
#, python-format
msgid "ambiguous option: %(option)s could match %(matches)s"
msgstr "option ambiguë: %(option)s pourrait correspondre à %(matches)s"

#: ../duplicity/argparse311.py:2207
#, python-format
msgid "unexpected option string: %s"
msgstr "chaîne d'option inattendue : %s"

#: ../duplicity/argparse311.py:2402
#, python-format
msgid "%r is not callable"
msgstr "%r n'est pas appelable"

#: ../duplicity/argparse311.py:2419
#, python-format
msgid "invalid %(type)s value: %(value)r"
msgstr "valeur %(type)s invalide : %(value)r"

#: ../duplicity/argparse311.py:2429
#, python-format
msgid "invalid choice: %(value)r (choose from %(choices)s)"
msgstr "choix invalide : %(value)r (choisir dans %(choices)s)"

#: ../duplicity/argparse311.py:2505
#, python-format
msgid "%(prog)s: error: %(message)s\n"
msgstr ""

#: ../duplicity/backend.py:201
#, python-format
msgid "Could not initialize backend: %s"
msgstr "Impossible d'initialiser le moteur : %s"

#: ../duplicity/backend.py:392
#, python-format
msgid "Backtrace of previous error: %s"
msgstr "Trace inverse de l'erreur précédente : %s"

#: ../duplicity/backend.py:413
#, python-format
msgid "Giving up after %s attempts. %s: %s. (for trace back: set log level DEBUG)"
msgstr ""

#: ../duplicity/backend.py:422
#, python-format
msgid "Giving up after %s attempts. %s: %s"
msgstr "Abandonner après %s tentatives. %s: %s"

#: ../duplicity/backend.py:428
#, python-format
msgid "Attempt of %s Nr. %s failed. %s: %s"
msgstr "La tentative de %s Nr. %s a échoué. %s: %s"

#: ../duplicity/backend.py:535
#, python-format
msgid "Reading results of '%s'"
msgstr "Lecture des résultats de « %s »"

#: ../duplicity/backend.py:562
#, python-format
msgid "Writing %s"
msgstr "Écriture de %s"

#: ../duplicity/backend.py:617
#, python-format
msgid "File %s not found locally after get from backend"
msgstr "Le fichier %s est introuvable localement après avoir été récupéré depuis le back-end"

#: ../duplicity/backend.py:670
#, python-format
msgid "%s Remote filesize %d for %s does not match local size %d"
msgstr ""

#: ../duplicity/backends/_testbackend.py:219
#, python-format
msgid "Unexpected exception while validate %s."
msgstr ""

#: ../duplicity/backends/giobackend.py:108
#, python-format
msgid "Connection failed, please check your password: %s"
msgstr "Connexion échouée, veuillez vérifier votre mot de passe : %s"

#: ../duplicity/backends/multibackend.py:100
#, python-format
msgid "MultiBackend: Could not parse query string %s: %s "
msgstr "MultiBackend : Impossible d'analyser la chaîne de requêtes %s : %s "

#: ../duplicity/backends/multibackend.py:110
#, python-format
msgid "MultiBackend: Invalid query string %s: more than one value for %s"
msgstr "MultiBackend : Chaîne de requêtes %s non valide : plus d'une valeur pour %s"

#: ../duplicity/backends/multibackend.py:116
#, python-format
msgid "MultiBackend: Invalid query string %s: unknown parameter %s"
msgstr "MultiBackend : Chaîne de requête %sinvalide : paramètre inconnu %s"

#: ../duplicity/backends/multibackend.py:167
#: ../duplicity/backends/multibackend.py:174
#, python-format
msgid "MultiBackend: illegal value for %s: %s"
msgstr "Multi-Backend : valeur illégale pour %s: %s"

#: ../duplicity/backends/multibackend.py:186
#, python-format
msgid "MultiBackend: Url %s"
msgstr "Multi-Backend : Url %s"

#: ../duplicity/backends/multibackend.py:189
#, python-format
msgid "MultiBackend: Could not load config file %s: %s "
msgstr "MultiBackend : Impossible de charger le fichier de configuration %s : %s "

#: ../duplicity/backends/multibackend.py:196
#, python-format
msgid "MultiBackend: use store %s"
msgstr "MultiBackend : utiliser le magasin de sauvegarde %s"

#: ../duplicity/backends/multibackend.py:200
#, python-format
msgid "MultiBackend: set env %s = %s"
msgstr "Multi-Backend: env %s = %s"

#: ../duplicity/backends/multibackend.py:214
#, python-format
msgid "Multibackend: register affinity for prefix %s"
msgstr "Multi-backend : enregistrer l'affinité pour le préfixe %s"

#: ../duplicity/backends/multibackend.py:257
#, python-format
msgid "MultiBackend: _put: write to store #%s (%s)"
msgstr "MultiBackend : _put: écrire dans le magasin #%s (%s)"

#: ../duplicity/backends/multibackend.py:272
#, python-format
msgid "MultiBackend: failed to write to store #%s (%s), try #%s, Exception: %s"
msgstr "MultiBackend : échec de l'écriture dans le magasin de sauvegarde #%s (%s), essayez #%s, exception : %s"

#: ../duplicity/backends/multibackend.py:286
#, python-format
msgid "MultiBackend: failed to write %s. Aborting process."
msgstr "MultiBackend : échec d'écriture %s. Abandon du processus."

#: ../duplicity/backends/multibackend.py:294
#, python-format
msgid "MultiBackend: failed to write %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend : échec de l'écriture de %s. Essais faits dans tous les magasins de sauvegarde, aucun n'a fonctionné"

#: ../duplicity/backends/multibackend.py:315
#, python-format
msgid "MultiBackend: failed to get %s to %s from %s"
msgstr "MutiBackend : impossible de mettre %s sur %s à partir de %s"

#: ../duplicity/backends/multibackend.py:320
#, python-format
msgid "MultiBackend: failed to get %s. Tried all backing stores and none succeeded"
msgstr "MutiBackend : impossible d'obtenir %s. Essais faits dans tous les magasins de sauvegarde, aucun n'a fonctionné"

#: ../duplicity/backends/multibackend.py:330
#, python-format
msgid "MultiBackend: %s: %d files"
msgstr "Multi-Backend : %s: %d fichiers"

#: ../duplicity/backends/multibackend.py:342
#, python-format
msgid "MultiBackend: combined list: %s"
msgstr "MultiBackend : liste combinée : %s"

#: ../duplicity/backends/multibackend.py:371
#: ../duplicity/backends/multibackend.py:398
#, python-format
msgid "MultiBackend: failed to delete %s. Tried all backing stores and none succeeded"
msgstr "MultiBackend : la supression de %s a échoué. Essais faits dans tous les magasins de sauvegarde, aucun n'a réussi"

#: ../duplicity/backends/onedrivebackend.py:88
#, python-format
msgid "Attempt of initialize_oauth2_session Nr. %s failed. %s: %s"
msgstr "La tentative d'initialize_oauth2_session n° %s a échoué. %s: %s"

#: ../duplicity/backends/pydrivebackend.py:203
#, python-format
msgid "PyDrive backend: multiple files called '%s'."
msgstr "Arrière-plan PyDrive : plusieurs fichiers nommés « %s »."

#: ../duplicity/backends/webdavbackend.py:68
msgid "Missing socket or ssl python modules."
msgstr "Il manque des modules de socket ou de python ssl."

#: ../duplicity/backends/webdavbackend.py:88
#, python-format
msgid "Cacert database file '%s' is not readable."
msgstr "Le fichier Cacert '%s' n'est pas lisible."

#: ../duplicity/backends/webdavbackend.py:153
#, python-format
msgid "Using WebDAV host %s port %s"
msgstr "Utilisation de l'hôte WebDAV %s port %s"

#: ../duplicity/backends/webdavbackend.py:154
#, python-format
msgid "Using WebDAV directory %s"
msgstr "Using WebDAV directory %s"

#: ../duplicity/backends/webdavbackend.py:184
#, python-format
msgid "WebDAV create connection on '%s'"
msgstr "WebDAV crée une connexion sur '%s'"

#: ../duplicity/backends/webdavbackend.py:192
#, python-format
msgid "WebDAV Unknown URI scheme: %s"
msgstr "Schéma URI inconnu WebDAV : %s"

#: ../duplicity/backends/webdavbackend.py:222
#, python-format
msgid "WebDAV %s %s request with headers: %s "
msgstr "Requête WebDAV %s %s avec en-têtes : %s "

#: ../duplicity/backends/webdavbackend.py:223
#: ../duplicity/backends/webdavbackend.py:246
#, python-format
msgid "WebDAV data length: %s "
msgstr "Longueur des données WebDAV : %s "

#: ../duplicity/backends/webdavbackend.py:226
#, python-format
msgid "WebDAV response status %s with reason '%s'."
msgstr "Statut de réponse WebDAV %s avec raison '%s'."

#: ../duplicity/backends/webdavbackend.py:232
#, python-format
msgid "WebDAV redirect to: %s "
msgstr "Redirection WebDAV vers : %s "

#: ../duplicity/backends/webdavbackend.py:234
msgid "WebDAV redirected 10 times. Giving up."
msgstr "WebDAV a redirigé 10 fois. Donne."

#: ../duplicity/backends/webdavbackend.py:239
msgid "WebDAV missing location header in redirect response."
msgstr "En-tête de localisation pour WebDAV manquant dans la réponse de redirection."

#: ../duplicity/backends/webdavbackend.py:244
msgid "WebDAV retry request with authentification headers."
msgstr "Requête WebDAV avec les en-têtes d'authentification."

#: ../duplicity/backends/webdavbackend.py:245
#, python-format
msgid "WebDAV %s %s request2 with headers: %s "
msgstr "Requête WebDAV %s %s avec en-têtes : %s "

#: ../duplicity/backends/webdavbackend.py:249
#, python-format
msgid "WebDAV response2 status %s with reason '%s'."
msgstr "Statut WebDAV response2 %s pour la raison «%s»."

#: ../duplicity/backends/webdavbackend.py:268
msgid "python-kerberos needed to use kerberos                           authorization, falling back to basic auth."
msgstr "python-kerberos a besoin d'utiliser l'autorisation kerberos, tombant à l'authentification de base."

#: ../duplicity/backends/webdavbackend.py:276
#, python-format
msgid "Kerberos authorization failed: %s.                          Falling back to basic auth."
msgstr "Échec de l’autorisation Kerberos : %s. Retour à l’authentification basique."

#: ../duplicity/backends/webdavbackend.py:377
#, python-format
msgid "Creating missing directory %s"
msgstr "Création du répertoire %s manquant"

#: ../duplicity/backends/webdavbackend.py:381
#, python-format
msgid "WebDAV MKCOL %s failed: %s %s"
msgstr "WebDAV MKCOL %s a échoué : %s %s"

#: ../duplicity/backends/webdavbackend.py:393
#, python-format
msgid "WebDAV path decoding and translation: %s -> %s"
msgstr "Décodage et traduction du chemin WebDAV : %s -> %s"

#: ../duplicity/backends/webdavbackend.py:429
#, python-format
msgid "WebDAV GET Bad status code %s reason %s."
msgstr ""

#: ../duplicity/backends/webdavbackend.py:450
#, python-format
msgid "WebDAV PUT Bad status code %s reason %s."
msgstr "WebDAV PUT Mauvais code de statut %s raison %s."

#: ../duplicity/backends/webdavbackend.py:469
#, python-format
msgid "WebDAV DEL Bad status code %s reason %s."
msgstr "WebDAV DEL Bad status code %s reason %s."

#. Used in usage help to represent a Unix-style path name. Example:
#. --archive-dir <path>
#: ../duplicity/cli_data.py:103 ../duplicity/cli_data.py:173
#: ../duplicity/cli_data.py:207 ../duplicity/cli_data.py:255
#: ../duplicity/cli_data.py:358 ../duplicity/cli_data.py:387
#: ../duplicity/cli_data.py:558 ../duplicity/cli_data.py:750
#: ../duplicity/cli_data.py:933
msgid "path"
msgstr "chemin"

#: ../duplicity/cli_data.py:116
msgid "Hot|Cool|Cold|Archive"
msgstr ""

#. Used in usage help to represent a desired number of
#. something. Example:
#. --num-retries <number>
#: ../duplicity/cli_data.py:121 ../duplicity/cli_data.py:127
#: ../duplicity/cli_data.py:135 ../duplicity/cli_data.py:163
#: ../duplicity/cli_data.py:179 ../duplicity/cli_data.py:457
#: ../duplicity/cli_data.py:477 ../duplicity/cli_data.py:524
#: ../duplicity/cli_data.py:546 ../duplicity/cli_data.py:552
#: ../duplicity/cli_data.py:570 ../duplicity/cli_data.py:641
#: ../duplicity/cli_data.py:649 ../duplicity/cli_data.py:701
#: ../duplicity/cli_data.py:787 ../duplicity/cli_data.py:920
msgid "number"
msgstr "nombre"

#. Used in usage help. Example:
#. --timeout <seconds>
#: ../duplicity/cli_data.py:147 ../duplicity/cli_data.py:757
#: ../duplicity/cli_data.py:946
msgid "seconds"
msgstr "secondes"

#. Used in usage help to represent an ID for a GnuPG key. Example:
#. --encrypt-key <gpg_key_id>
#: ../duplicity/cli_data.py:195 ../duplicity/cli_data.py:201
#: ../duplicity/cli_data.py:381 ../duplicity/cli_data.py:708
#: ../duplicity/cli_data.py:912
msgid "gpg-key-id"
msgstr "identifiant_clé_gpg"

#. Used in usage help to represent a "glob" style pattern for
#. matching one or more files, as described in the documentation.
#. Example:
#. --exclude <shell_pattern>
#: ../duplicity/cli_data.py:212 ../duplicity/cli_data.py:409
#: ../duplicity/cli_data.py:951
msgid "shell_pattern"
msgstr "format@@0 shell_pattern"

#. Used in usage help to represent the name of a file. Example:
#. --log-file <filename>
#: ../duplicity/cli_data.py:224 ../duplicity/cli_data.py:230
#: ../duplicity/cli_data.py:291 ../duplicity/cli_data.py:415
#: ../duplicity/cli_data.py:909
msgid "filename"
msgstr "nom_de_fichier"

#. Used in usage help to represent a time spec for a previous
#. point in time, as described in the documentation. Example:
#. duplicity remove-older-than time [options] target_url
#: ../duplicity/cli_data.py:236 ../duplicity/cli_data.py:352
#: ../duplicity/cli_data.py:583 ../duplicity/cli_data.py:977
msgid "time"
msgstr "temps"

#: ../duplicity/cli_data.py:249 ../duplicity/cli_data.py:421
msgid "regex"
msgstr "regex"

#: ../duplicity/cli_data.py:404
msgid "imap_mailbox"
msgstr "boîte_aux_lettres_IMAP"

#: ../duplicity/cli_data.py:429
msgid "If set, an extra file with statistics in json format will be stored with each backup. It includes the statistics printed out on stdout by default. With 'collection-status' it returs these stats."
msgstr "Si défini, un fichier supplémentaire avec des statistiques au format json sera stocké à chaque sauvegarde. Il inclut les statistiques imprimées sur stdout par défaut. Avec \"collection-status\", il retarde ces statistiques."

#: ../duplicity/cli_data.py:437
msgid "file_descriptor"
msgstr "descripteur_fichier"

#: ../duplicity/cli_data.py:444
msgid "log_filename"
msgstr "Nom du fichier journal"

#: ../duplicity/cli_data.py:483
msgid "backup name"
msgstr "nom_sauvegarde"

#: ../duplicity/cli_data.py:600
msgid "s3_endpoint_url"
msgstr "s3_endpoint_url"

#: ../duplicity/cli_data.py:668
msgid "s3_kms_key_id"
msgstr "s3_kms_key_id"

#: ../duplicity/cli_data.py:674
msgid "s3_kms_grant"
msgstr "S3_kms_accorder"

#: ../duplicity/cli_data.py:680
msgid "s3_region_name"
msgstr "format@@0 s3_region_name"

#: ../duplicity/cli_data.py:686
msgid "policy"
msgstr "politique"

#. noun
#: ../duplicity/cli_data.py:691 ../duplicity/cli_data.py:696
#: ../duplicity/cli_data.py:898
msgid "command"
msgstr "commande"

#: ../duplicity/cli_data.py:736
msgid "pem formatted bundle of certificate authorities"
msgstr "Convertir au format PEM l'ensemble des autorités de certification"

#: ../duplicity/cli_data.py:741
msgid "path to a folder with certificate authority files"
msgstr "chemin vers un dossier avec des fichiers d'autorité de certificat"

#. abbreviation for "character" (noun)
#: ../duplicity/cli_data.py:763 ../duplicity/cli_data.py:896
msgid "char"
msgstr "caract."

#: ../duplicity/cli_data.py:776
msgid "verb"
msgstr "verbe"

#: ../duplicity/cli_data.py:794
msgid "extra headers for Webdav, like 'Cookie,name=value'"
msgstr "en-têtes supplémentaires pour Webdav, comme 'Cookie,name=value'"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other_host[:port]//absolute_path
#: ../duplicity/cli_data.py:887
msgid "absolute_path"
msgstr "chemin_absolu"

#. Used in usage help. Example:
#. tahoe://alias/some_dir
#: ../duplicity/cli_data.py:890
msgid "alias"
msgstr "alias"

#. Used in help to represent a "bucket name" for Amazon Web
#. Services' Simple Storage Service (S3). Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:894
msgid "bucket_name"
msgstr "nom_de_bucket"

#. Used in usage help to represent the name of a container in
#. Amazon Web Services' Cloudfront. Example:
#. cf+http://container_name
#: ../duplicity/cli_data.py:902
msgid "container_name"
msgstr "nom_de_conteneur"

#. noun
#: ../duplicity/cli_data.py:904
msgid "count"
msgstr "compte"

#. Used in usage help to represent the name of a file directory
#: ../duplicity/cli_data.py:906
msgid "directory"
msgstr "dossier"

#. Used in usage help, e.g. to represent the name of a code
#. module. Example:
#. rsync://user[:password]@other.host[:port]::/module/some_dir
#: ../duplicity/cli_data.py:916
msgid "module"
msgstr "module"

#. Used in usage help to represent an internet hostname. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:927
msgid "other.host"
msgstr "autre.hote"

#. Used in usage help. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:930
msgid "password"
msgstr "mot_de_passe"

#. Used in usage help to represent a TCP port number. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:936
msgid "port"
msgstr "port"

#. Used in usage help. This represents a string to be used as a
#. prefix to names for backup files created by Duplicity. Example:
#. s3://other.host/bucket_name[/prefix]
#: ../duplicity/cli_data.py:940
msgid "prefix"
msgstr "préfixe"

#. Used in usage help to represent a Unix-style path name. Example:
#. rsync://user[:password]@other.host[:port]/relative_path
#: ../duplicity/cli_data.py:943
msgid "relative_path"
msgstr "chemin_relatif"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. Example:
#. file:///some_dir
#: ../duplicity/cli_data.py:955
msgid "some_dir"
msgstr "dossier_quelconque"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory where files will be
#. coming FROM. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:960
msgid "source_path"
msgstr "chemin de la source"

#. Used in usage help to represent a URL files will be coming
#. FROM. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:964
msgid "source_url"
msgstr "url_source"

#. Used in usage help to represent the name of a single file
#. directory or a Unix-style path to a directory. where files will be
#. going TO. Example:
#. duplicity [restore] [options] source_url target_dir
#: ../duplicity/cli_data.py:969
msgid "target_dir"
msgstr "dossier_cible"

#. Used in usage help to represent a URL files will be going TO.
#. Example:
#. duplicity [full|incremental] [options] source_path target_url
#: ../duplicity/cli_data.py:973
msgid "target_url"
msgstr "url_cible"

#. Used in usage help to represent a user name (i.e. login).
#. Example:
#. ftp://user[:password]@other.host[:port]/some_dir
#: ../duplicity/cli_data.py:981
msgid "user"
msgstr "utilisateur"

#. account id for b2. Example: b2://account_id@bucket/
#: ../duplicity/cli_data.py:983
msgid "account_id"
msgstr "identifiant_de_compte"

#. application_key for b2.
#. Example: b2://account_id:application_key@bucket/
#: ../duplicity/cli_data.py:986
msgid "application_key"
msgstr "clef_d_application"

#. remote name for rclone.
#. Example: rclone://remote:/some_dir
#: ../duplicity/cli_data.py:989 ../duplicity/dup_collections.py:564
msgid "remote"
msgstr "distant"

#: ../duplicity/cli_data.py:993
msgid "Backends and their URL formats:"
msgstr "Moteurs et leurs formats URL :"

#: ../duplicity/cli_main.py:151
msgid "No valid action found. Will imply 'backup' because a path source was given and target is a url location."
msgstr "Aucune action valide trouvée. Inclura 'backup' car une source de chemin a été donnée et la cible est un emplacement url."

#: ../duplicity/cli_main.py:159
msgid "No valid action found. Will imply 'restore' because url source was given and target is a local path."
msgstr "Aucune action valide trouvée. Cela impliquera 'restorer' car la source de l'URL a été donnée et la cible est un chemin local."

#: ../duplicity/cli_util.py:45
msgid "Enter 'duplicity --help' for help screen."
msgstr "Entrez 'duplicity --help' pour l'écran d'aide."

#: ../duplicity/cli_util.py:127
msgid "Running in 'ignore errors' mode due to --ignore-errors.\n"
"Please reconsider if this was not intended"
msgstr "Exécution en mode 'ignore les erreurs' due à --ignore-errors.\n"
"Veuillez reconsidérer si ce n'était pas prévu"

#: ../duplicity/cli_util.py:281
msgid "Verbosity must be one of: digit [0-9], character [ewnid],\n"
"or word ['error', 'warning', 'notice', 'info', 'debug'].\n"
"The default is 3 (Notice).  It is strongly recommended\n"
"that verbosity level is set at 2 (Warning) or higher."
msgstr ""

#: ../duplicity/diffdir.py:105 ../duplicity/diffdir.py:404
#, python-format
msgid "Error %s getting delta for %s"
msgstr "Erreur %s pour l'obtention du delta pour %s"

#: ../duplicity/diffdir.py:119
#, python-format
msgid "Getting delta of %s and %s"
msgstr "Obtention du delta de %s et %s"

#: ../duplicity/diffdir.py:163
#, python-format
msgid "A %s"
msgstr ""

#: ../duplicity/diffdir.py:171
#, python-format
msgid "M %s"
msgstr "M %s"

#: ../duplicity/diffdir.py:194
#, python-format
msgid "Comparing %s and %s"
msgstr "Comparaison de %s et %s"

#: ../duplicity/diffdir.py:207
#, python-format
msgid "D %s"
msgstr ""

#: ../duplicity/dup_collections.py:199 ../duplicity/dup_collections.py:208
#, python-format
msgid "BackupSet.delete: missing %s"
msgstr "BackupSet.delete : %s manquant"

#: ../duplicity/dup_collections.py:234
msgid "Fatal Error: No manifests found for most recent backup"
msgstr "Erreur fatale : aucun fichier manifest présent pour la sauvegarde la plus récente"

#: ../duplicity/dup_collections.py:247
msgid "Fatal Error: Remote manifest does not match local one.  Either the remote backup set or the local archive directory has been corrupted."
msgstr "Erreur fatale : le fichier manifest distant ne correspond pas à celui en local. Soit le jeu de sauvegarde distant, soit le dossier local d’archive a été corrompu."

#: ../duplicity/dup_collections.py:255
#, python-format
msgid "Error processing remote manifest (%s): %s"
msgstr "Erreur lors du traitement du manifeste distant (%s): %s"

#: ../duplicity/dup_collections.py:264
msgid "Fatal Error: Neither remote nor local manifest is readable."
msgstr "Erreur fatale : ni le fichier manifest local, ni le distant ne sont lisibles."

#: ../duplicity/dup_collections.py:275
#, python-format
msgid "Processing local manifest %s (%s)"
msgstr "Traitement du manifeste local %s (%s)"

#: ../duplicity/dup_collections.py:310
msgid "No Jsonstat file found, return empty."
msgstr ""

#: ../duplicity/dup_collections.py:403
msgid "Preferring Backupset over previous one!"
msgstr "Jeu de sauvegarde préféré au précédent !"

#: ../duplicity/dup_collections.py:407
#, python-format
msgid "Ignoring incremental Backupset (start_time: %s; needed: %s)"
msgstr "Ignore le jeu de sauvegarde incrémentale (date_début : %s ; nécessaire : %s)"

#: ../duplicity/dup_collections.py:416
#, python-format
msgid "Added incremental Backupset (start_time: %s / end_time: %s)"
msgstr "Jeu de sauvegarde incrémentale ajouté (date_début : %s ; date_fin : %s)"

#: ../duplicity/dup_collections.py:489
msgid "Chain start time: "
msgstr "Date de début de chaîne : "

#: ../duplicity/dup_collections.py:490
msgid "Chain end time: "
msgstr "Date de fin de chaîne : "

#: ../duplicity/dup_collections.py:491
#, python-format
msgid "Number of contained backup sets: %d"
msgstr "Nombre de jeux de sauvegarde contenus : %d"

#: ../duplicity/dup_collections.py:492
#, python-format
msgid "Total number of contained volumes: %d"
msgstr "Nombre total de volumes contenus : %d"

#: ../duplicity/dup_collections.py:493 ../duplicity/dup_collections.py:1299
msgid "Type of backup set:"
msgstr "Type de jeu de sauvegarde :"

#: ../duplicity/dup_collections.py:493 ../duplicity/dup_collections.py:1299
msgid "Time:"
msgstr "Date :"

#: ../duplicity/dup_collections.py:493
msgid "Num volumes:"
msgstr "Nombre de volumes :"

#: ../duplicity/dup_collections.py:498 ../duplicity/dup_collections.py:1306
msgid "Full"
msgstr "Complète"

#: ../duplicity/dup_collections.py:501 ../duplicity/dup_collections.py:1308
msgid "Incremental"
msgstr "Incrémentale"

#: ../duplicity/dup_collections.py:562
msgid "local"
msgstr "local"

#: ../duplicity/dup_collections.py:724
msgid "Collection Status"
msgstr "État de la collection"

#: ../duplicity/dup_collections.py:726
#, python-format
msgid "Connecting with backend: %s"
msgstr "Connexion au serveur central : %s"

#: ../duplicity/dup_collections.py:727
#, python-format
msgid "Archive dir: %s"
msgstr "Dossier d’archive : %s"

#: ../duplicity/dup_collections.py:730
#, python-format
msgid "Found %d secondary backup chain(s)."
msgstr "Chaîne(s) de sauvegarde secondaire %d trouvée."

#: ../duplicity/dup_collections.py:732
#, python-format
msgid "Secondary chain %d of %d:"
msgstr "Chaîne secondaire %d sur %d :"

#: ../duplicity/dup_collections.py:737
msgid "Found primary backup chain with matching signature chain:"
msgstr "Chaîne primaire de sauvegarde trouvée avec la signature de chaîne correspondante :"

#: ../duplicity/dup_collections.py:740
msgid "No backup chains with active signatures found"
msgstr "Aucune chaîne de sauvegarde avec les signatures actives n’a été trouvée"

#: ../duplicity/dup_collections.py:743
#, python-format
msgid "Also found %d backup set(s) not part of any chain,"
msgstr "Également trouvé un(des) ensemble(s) de sauvegarde %d ne fait partie d'aucune chaîne,"

#: ../duplicity/dup_collections.py:744
#, python-format
msgid "and %d incomplete backup set(s)."
msgstr "et un/des ensembles de sauvegarde incomplet(s) %d."

#. "cleanup" is a hard-coded command, so do not translate it
#: ../duplicity/dup_collections.py:746
msgid "These may be deleted by running duplicity with the \"cleanup\" command."
msgstr "Ceux-ci peuvent être supprimés en exécutant duplicity avec la commande « cleanup »."

#: ../duplicity/dup_collections.py:748
msgid "No orphaned or incomplete backup sets found."
msgstr "Aucun jeu orphelin ou incomplet de sauvegarde n’a été trouvé."

#: ../duplicity/dup_collections.py:764
#, python-format
msgid "%d file(s) exists on backend"
msgstr "%d fichier(s) existe sur le backend"

#: ../duplicity/dup_collections.py:768
#, python-format
msgid "%d file(s) exist in cache"
msgstr ""

#: ../duplicity/dup_collections.py:822
msgid "Warning, discarding last backup set, because of missing signature file."
msgstr "Avertissement, rejet du dernier jeu de sauvegarde à cause de l’absence du fichier de signatures."

#: ../duplicity/dup_collections.py:845
msgid "Warning, found the following local orphaned signature file(s):"
msgstr "Attention, les fichiers de signature locaux orphelins suivants ont été trouvés :"

#: ../duplicity/dup_collections.py:853
msgid "Warning, found the following remote orphaned signature file(s):"
msgstr "Attention, les fichiers de signature distants orphelins suivants ont été trouvés :"

#: ../duplicity/dup_collections.py:861
msgid "Warning, found signatures but no corresponding backup files"
msgstr "Avertissement, signatures trouvées mais aucun fichier de sauvegarde correspondant"

#: ../duplicity/dup_collections.py:867
msgid "Warning, found incomplete backup sets, probably left from aborted session"
msgstr "Avertissement, jeux de sauvegarde incomplets trouvés, probablement laissés par des sessions interrompues"

#: ../duplicity/dup_collections.py:873
msgid "Warning, found the following orphaned backup file(s):"
msgstr "Attention, les fichiers de sauvegarde orphelins suivants ont été trouvés :"

#: ../duplicity/dup_collections.py:888
#, python-format
msgid "Extracting backup chains from list of files: %s"
msgstr "Extraction des chaînes de sauvegarde depuis la liste des fichiers : %s"

#: ../duplicity/dup_collections.py:899
#, python-format
msgid "File %s is part of known set"
msgstr "Le fichier %s fait partie d’un jeu connu"

#: ../duplicity/dup_collections.py:902
#, python-format
msgid "File %s is not part of a known set; creating new set"
msgstr "Le fichier %s ne fait pas partie d’un jeu connu ; création d’un nouveau jeu"

#: ../duplicity/dup_collections.py:907
#, python-format
msgid "Ignoring file (rejected by backup set) '%s'"
msgstr "Fichier ignoré (rejeté par le jeu de sauvegarde) « %s »"

#: ../duplicity/dup_collections.py:923
#, python-format
msgid "Found backup chain %s"
msgstr "Chaîne de sauvegarde %s trouvée"

#: ../duplicity/dup_collections.py:928
#, python-format
msgid "Added set %s to pre-existing chain %s"
msgstr "Jeu %s ajouté à la chaîne préexistante %s"

#: ../duplicity/dup_collections.py:931
#, python-format
msgid "Found orphaned set %s"
msgstr "Jeu orphelin %s trouvé"

#: ../duplicity/dup_collections.py:1079
#, python-format
msgid "No signature chain for the requested time. Using oldest available chain, starting at time %s."
msgstr "Aucune chaîne de signature pour l'heure demandée. Utilisation de la plus ancienne chaîne disponible, à partir de l'heure %s."

#: ../duplicity/dup_collections.py:1297
#, python-format
msgid "File: %s"
msgstr "Fichier : %s"

#: ../duplicity/dup_collections.py:1298
#, python-format
msgid "Total number of backup: %d"
msgstr "Nombre total de sauvegardes : %d"

#: ../duplicity/dup_collections.py:1299 ../duplicity/dup_collections.py:1335
msgid "Type of file change:"
msgstr "Type de changement de fichier :"

#: ../duplicity/dup_collections.py:1333
#, python-format
msgid " Backup set time: %s"
msgstr " Heure de sauvegarde définie : %s"

#: ../duplicity/dup_collections.py:1334
#, python-format
msgid "Total number of changes: %d"
msgstr "Nombre total de modifications : %d"

#: ../duplicity/dup_collections.py:1335
msgid "File:"
msgstr "Fichier :"

#: ../duplicity/dup_main.py:111
msgid "Reuse configured PASSPHRASE as SIGN_PASSPHRASE"
msgstr "Réutiliser la PASSPHRASE configurée comme SIGN_PASSPHRASE"

#: ../duplicity/dup_main.py:122
msgid "Reuse configured SIGN_PASSPHRASE as PASSPHRASE"
msgstr "Réutiliser la SIGN_PASSPHRASE configurée comme PASSPHRASE"

#: ../duplicity/dup_main.py:160
msgid "Skipping passphrase input for full backup with encryption keys."
msgstr ""

#: ../duplicity/dup_main.py:165
msgid "PASSPHRASE variable not set, asking user."
msgstr "Variable PASSPHRASE non établie, demande à l'utilisateur."

#: ../duplicity/dup_main.py:180
msgid "GnuPG passphrase for signing key:"
msgstr ""

#: ../duplicity/dup_main.py:185
msgid "GnuPG passphrase for decryption:"
msgstr ""

#: ../duplicity/dup_main.py:190
msgid "Retype passphrase for signing key to confirm: "
msgstr "Veuillez retaper la phrase de passe pour la clé de signature pour confirmer : "

#: ../duplicity/dup_main.py:192
msgid "Retype passphrase for decryption to confirm: "
msgstr "Retapez le mot de passe pour le décryptage pour confirmer: "

#: ../duplicity/dup_main.py:196
msgid "First and second passphrases do not match!  Please try again."
msgstr "La première et la seconde phrases de passe ne correspondent pas ! Veuillez réessayer."

#: ../duplicity/dup_main.py:207
msgid "Cannot use empty passphrase with symmetric encryption!  Please try again."
msgstr "Impossible d'utiliser une phrase de passe vide avec un chiffrement symétrique ! Veuillez réessayer."

#: ../duplicity/dup_main.py:267
#, python-format
msgid "File %s complete in backup set.\n"
"Continuing restart on file %s."
msgstr "Le fichier %s est terminé dans le jeu de sauvegarde.\n"
"Le redémarrage continue sur fichier %s."

#: ../duplicity/dup_main.py:277
#, python-format
msgid "File %s missing in backup set.\n"
"Continuing restart on file %s."
msgstr "Le fichier %s est manquant dans le jeu de sauvegarde.\n"
"Le redémarrage continue sur le fichier %s."

#: ../duplicity/dup_main.py:332
#, python-format
msgid "File %s was corrupted during upload."
msgstr "Le fichier % s a été endommagé pendant le téléversement."

#: ../duplicity/dup_main.py:362
msgid "Restarting backup, but current encryption settings do not match original settings"
msgstr "Redémarrage de sauvegarde, mais les paramètres de chiffrement actuels ne correspondent pas aux paramètres originaux"

#: ../duplicity/dup_main.py:385 ../duplicity/dup_main.py:521
#, python-format
msgid "Processed volume %d"
msgstr "Volume %d traité"

#: ../duplicity/dup_main.py:433
msgid "Skipping encryption validation due to glacier/deep storage"
msgstr "Ignorer la validation du chiffrement en raison du stockage glacier/profond"

#: ../duplicity/dup_main.py:437
#, python-format
msgid "Restarting after volume %s, file %s, block %s"
msgstr "Redémarrage après le volume %s, le fichier %s, le bloc %s"

#: ../duplicity/dup_main.py:489
msgid "Skipped volume upload, as effectivly nothing has changed"
msgstr "Envoi de volume ignoré, car effectivement, rien n'a changé"

#: ../duplicity/dup_main.py:712
msgid "Fatal Error: Unable to start incremental backup.  Old signatures not found and incremental specified"
msgstr "Erreur fatale : impossible de démarrer la sauvegarde incrémentale. Les anciennes signatures sont introuvables pour la méthode incrémentale indiquée"

#: ../duplicity/dup_main.py:718
msgid "No signatures found, switching to full backup."
msgstr "Aucune signature de sauvegarde trouvée, bascule vers une sauvegarde complète."

#: ../duplicity/dup_main.py:732
msgid "Backup Statistics"
msgstr "Statistiques de sauvegarde"

#: ../duplicity/dup_main.py:872
#, python-format
msgid "%s not found in archive - no files restored."
msgstr "%s introuvable dans l'archive - aucun fichier n'a été restauré."

#: ../duplicity/dup_main.py:876
msgid "No files found in archive - nothing restored."
msgstr "Aucun fichier trouvé dans l’archive – rien n'a été restauré."

#: ../duplicity/dup_main.py:914
#, python-format
msgid "Processed volume %d of %d"
msgstr "Le volume %d sur %d a été traité"

#: ../duplicity/dup_main.py:957
#, python-format
msgid "Invalid data - %s hash mismatch for file:"
msgstr "Données non valides - empreinte %s non vérifiée pour le fichier :"

#: ../duplicity/dup_main.py:959
#, python-format
msgid "Calculated hash: %s"
msgstr "Empreinte calculée : %s"

#: ../duplicity/dup_main.py:960
#, python-format
msgid "Manifest hash: %s"
msgstr "Empreinte du fichier manifest : %s"

#: ../duplicity/dup_main.py:967 ../duplicity/util.py:110
#, python-format
msgid "IGNORED_ERROR: Warning: ignoring error as requested: %s: %s"
msgstr "ERREUR_IGNORÉE : attention : ignore l'erreur comme demandé : %s : %s"

#: ../duplicity/dup_main.py:1014
#, python-format
msgid "Volume was signed by key %s, not %s"
msgstr "Le volume a été signé par la clé %s et non pas %s"

#: ../duplicity/dup_main.py:1046
#, python-format
msgid "Verify complete: %s, %s."
msgstr "Vérification complète : %s, %s."

#: ../duplicity/dup_main.py:1047
#, python-format
msgid "%d file(s) compared"
msgstr ""

#: ../duplicity/dup_main.py:1047
#, python-format
msgid "%d difference(s) found"
msgstr ""

#: ../duplicity/dup_main.py:1066
msgid "No extraneous files found, nothing deleted in cleanup."
msgstr "Aucun fichier étranger détecté, aucune suppression effectuée lors du nettoyage."

#: ../duplicity/dup_main.py:1071
msgid "Deleting these file(s) from backend:"
msgstr ""

#: ../duplicity/dup_main.py:1081
msgid "Found the following file(s) to delete:"
msgstr "Trouvé le(s) fichier(s) suivant(s) à supprimer :"

#: ../duplicity/dup_main.py:1085
msgid "Run duplicity again with the --force option to actually delete."
msgstr "Exécutez duplicity à nouveau avec l’option --force pour supprimer réellement."

#: ../duplicity/dup_main.py:1131
msgid "There are backup set(s) at time(s):"
msgstr "Il y a un ou plusieurs jeux de sauvegarde datés du :"

#: ../duplicity/dup_main.py:1133
msgid "Which can't be deleted because newer sets depend on them."
msgstr "Qui ne peuvent être supprimés car de nouveaux jeux en dépendent."

#: ../duplicity/dup_main.py:1140
msgid "Current active backup chain is older than specified time.  However, it will not be deleted.  To remove all your backups, manually purge the repository."
msgstr "La chaîne de sauvegarde active actuelle est plus ancienne que la date indiquée. Cependant elle ne sera pas supprimée. Pour ôter toutes vos sauvegardes, purgez manuellement leur dépôt."

#: ../duplicity/dup_main.py:1158
msgid "No old backup sets found, nothing deleted."
msgstr "Aucun ancien jeu de sauvegarde n’a été trouvé, rien n’a été supprimé."

#: ../duplicity/dup_main.py:1161
msgid "Deleting backup chain(s) at time:"
msgstr "Suppression de la (des) chaîne(s) de sauvegarde au moment :"

#: ../duplicity/dup_main.py:1170
#, python-format
msgid "Deleting any incremental signature chain rooted at %s"
msgstr "Suppression de toute chaîne de signature incrémentale enracinée à %s"

#: ../duplicity/dup_main.py:1172
#, python-format
msgid "Deleting any incremental backup chain rooted at %s"
msgstr "Suppression de toute chaîne de sauvegarde incrémentale rootée à %s"

#: ../duplicity/dup_main.py:1175
#, python-format
msgid "Deleting complete signature chain %s"
msgstr "Suppression de la chaîne de signature complète %s"

#: ../duplicity/dup_main.py:1177
#, python-format
msgid "Deleting complete backup chain %s"
msgstr "Suppression de la chaîne de sauvegarde complète %s"

#: ../duplicity/dup_main.py:1184
msgid "Found old backup chain(s) at the following time:"
msgstr "Trouvé d'anciennes chaînes de sauvegarde au moment suivant:"

#: ../duplicity/dup_main.py:1188
msgid "Rerun command with --force option to actually delete."
msgstr "Veuillez relancer la commande avec l'option --force pour forcer la suppression."

#: ../duplicity/dup_main.py:1291
#, python-format
msgid "Deleting local %s (not authoritative at backend)."
msgstr "Supprime le %s local (ne fait pas autorité au niveau du serveur)."

#: ../duplicity/dup_main.py:1295
#, python-format
msgid "Unable to delete %s: %s"
msgstr "Impossible de supprimer %s : %s"

#: ../duplicity/dup_main.py:1329 ../duplicity/dup_temp.py:296
#, python-format
msgid "Failed to read %s: %s"
msgstr "Impossible de lire %s : %s"

#: ../duplicity/dup_main.py:1341
#, python-format
msgid "Copying %s to local cache."
msgstr "Copie de %s vers le cache local."

#: ../duplicity/dup_main.py:1389
msgid "Local and Remote metadata are synchronized, no sync needed."
msgstr "Les métadonnées locales et distantes sont déjà synchronisées. Aucune synchronisation nécessaire."

#: ../duplicity/dup_main.py:1394
msgid "Synchronizing remote metadata to local cache..."
msgstr "Synchronisation des métadonnées distantes vers le cache local..."

#: ../duplicity/dup_main.py:1408
msgid "Sync would copy the following from remote to local:"
msgstr "La synchronisation va copier localement les éléments suivants à partir de l’hôte distant :"

#: ../duplicity/dup_main.py:1414
msgid "Sync would remove the following spurious local files:"
msgstr "La synchronisation va supprimer les fichiers parasites suivants :"

#: ../duplicity/dup_main.py:1460
msgid "Unable to get free space on temp."
msgstr "Impossible de déterminer l’espace libre dans le dossier temporaire."

#: ../duplicity/dup_main.py:1478
msgid "Unable to get max open files."
msgstr "Impossible d’obtenir le maximum de fichiers ouverts."

#: ../duplicity/dup_main.py:1484
msgid "Use 'ulimit -n 1024' or higher to correct.\n"
msgstr ""

#: ../duplicity/dup_main.py:1539
msgid "RESTART: The first volume failed to upload before termination.\n"
"         Restart is impossible...starting backup from beginning."
msgstr "REDÉMARRAGE : échec de téléchargement du premier volume avant le délai de fin.\n"
"              Le redémarrage est impossible... lancement de la sauvegarde depuis le début."

#: ../duplicity/dup_main.py:1549
#, python-format
msgid "RESTART: Volumes %d to %d failed to upload before termination.\n"
"         Restarting backup at volume %d."
msgstr "REDÉMARRAGE : échec de téléchargement de %d volumes sur %d avant le délai de fin.\n"
"              Redémarrage de la sauvegarde au volume %d."

#: ../duplicity/dup_main.py:1560
#, python-format
msgid "RESTART: Impossible backup state: manifest has %d vols, remote has %d vols.\n"
"         Restart is impossible ... duplicity will clean off the last partial\n"
"         backup then restart the backup from the beginning."
msgstr "REDÉMARRAGE : état de sauvegarde impossible :\n"
"              le fichier manifest indique %d volumes, le distant en a %d.\n"
"              Redémarrage impossible... duplicity va nettoyer la dernière\n"
"              sauvegarde partielle puis reprendre la sauvegarde depuis le début."

#: ../duplicity/dup_main.py:1586
msgid "                PYTHONOPTIMIZE in the environment causes duplicity to fail to\n"
"                recognize its own backups.  Please remove PYTHONOPTIMIZE from\n"
"                the environment and rerun the backup.\n\n"
"                See https://bugs.launchpad.net/duplicity/+bug/931175\n"
"                "
msgstr ""

#: ../duplicity/dup_main.py:1688
msgid "Last full backup date:"
msgstr ""

#: ../duplicity/dup_main.py:1690
msgid "Last full backup date: none"
msgstr "Date de la dernière sauvegarde complète : aucune"

#: ../duplicity/dup_main.py:1697
msgid "Last full backup is too old, forcing full backup"
msgstr "La dernière sauvegarde complète est trop ancienne, forçage d’une nouvelle sauvegarde complète"

#: ../duplicity/dup_main.py:1753
msgid "When using symmetric encryption, the signing passphrase must equal the encryption passphrase."
msgstr "Lors de l'utilisation du chiffrement symétrique, le mot de passe de signature doit correspondre au mot de passe de chiffrement."

#: ../duplicity/dup_time.py:54
#, python-format
msgid "Bad interval string \"%s\"\n\n"
"Intervals are specified like 2Y (2 years) or 2h30m (2.5 hours).  The\n"
"allowed special characters are s, m, h, D, W, M, and Y.  See the man\n"
"page for more information."
msgstr "Mauvais chaîne d'intervalle « %s »\n\n"
"Les intervalles sont spécifiés comme 2Y (2 ans) ou 2h30min (2,5 heures). Les\n"
"caractères spéciaux autorisés sont s, m, h, D, W, M et Y. Voir la page de manuel\n"
"pour plus d'informations."

#: ../duplicity/dup_time.py:63
#, python-format
msgid "Bad time string \"%s\"\n\n"
"The acceptible time strings are intervals (like \"3D64s\"), w3-datetime\n"
"strings, like \"2002-04-26T04:22:01-07:00\" (strings like\n"
"\"2002-04-26T04:22:01\" are also acceptable - duplicity will use the\n"
"current time zone), or ordinary dates like 2/4/1997 or 2001-04-23\n"
"(various combinations are acceptable, but the month always precedes\n"
"the day)."
msgstr "Mauvais chaîne de temps « %s »\n\n"
"Les chaînes de temps acceptées sont les intervalles (comme « 3D64s »), des chaînes\n"
"au format w3-datetime, comme « 2002-04-26T04:22:01-07:00 » (les chaînes comme\n"
"« 2002-04-26T04:22:01 » sont aussi acceptées - duplicity utilisera le \n"
"fuseau horaire courant), ou des dates ordinaires comme 2/4/1997 ou 2001-04-23\n"
"(plusieurs combinaisons sont possibles, mais le mois précède toujours le jour)."

#: ../duplicity/lazy.py:349
#, python-format
msgid "Warning: oldindex %s >= newindex %s"
msgstr "Attention : ancienindice %s >= nouvelindice %s"

#: ../duplicity/lazy.py:424
#, python-format
msgid "Error '%s' processing %s"
msgstr "Erreur « %s » lors du traitement de %s"

#: ../duplicity/lazy.py:436
#, python-format
msgid "Skipping %s because of previous error"
msgstr "Passe %s à cause d'une erreur précédente"

#: ../duplicity/librsync.py:194
msgid "basis_file must be a (true) file or an object whose file attribute is the underlying true file object"
msgstr "basis_file doit être un (vrai) fichier ou un objet dont l'attribut fichier est l'objet de fichier véritable sous-jacent"

#: ../duplicity/manifest.py:95
#, python-format
msgid "Fatal Error: Backup source host has changed.\n"
"Current hostname: %s\n"
"Previous hostname: %s"
msgstr "Erreur fatale : l'hôte de la source de sauvegarde a changé.\n"
"Nom d'hôte courant : %s\n"
"Nom d'hôte précédent : %s"

#: ../duplicity/manifest.py:102
msgid "Fatal Error: Backup source directory has changed.\n"
msgstr "Erreur fatale : Le répertoire source de la sauvegarde a changé.\n"

#: ../duplicity/manifest.py:115
msgid "Aborting because you may have accidentally tried to backup two different data sets to the same remote location, or using the same archive directory.  If this is not a mistake, use the --allow-source-mismatch switch to avoid seeing this message"
msgstr "Annulation parce que vous avez peut-être accidentellement essayé de sauvegarder deux jeux de données différents au même emplacement distant ou utilisé le même dossier d'archives. Si ceci est une erreur, utilisez l'option --allow-source-mismatch pour éviter de voir ce message."

#: ../duplicity/manifest.py:221
#, python-format
msgid "Found manifest volume %s"
msgstr "Volume manifeste %s trouvé"

#: ../duplicity/manifest.py:228
#, python-format
msgid "Found %s volumes in manifest"
msgstr "Nombre %s trouvé dans le manifeste"

#: ../duplicity/manifest.py:268
msgid "Manifests not equal because different volume numbers"
msgstr "Fichiers manifest non égaux car les numéros de volume diffèrent"

#: ../duplicity/manifest.py:273
msgid "Manifests not equal because volume lists differ"
msgstr "Fichiers manifest non égaux car les listes de volume diffèrent"

#: ../duplicity/manifest.py:277
msgid "Manifests not equal because hosts or directories differ"
msgstr "Fichiers manifest non égaux car les hôtes ou les répertoires diffèrent"

#: ../duplicity/manifest.py:438
msgid "Warning, found extra Volume identifier"
msgstr "Attention, un identifiant de volume supplémentaire a été trouvé"

#: ../duplicity/manifest.py:464
msgid "Other is not VolumeInfo"
msgstr "Autre n'est pas VolumeInfo"

#: ../duplicity/manifest.py:467
msgid "Volume numbers don't match"
msgstr "Les numéros de volume ne correspondent pas"

#: ../duplicity/manifest.py:470
msgid "start_indicies don't match"
msgstr "les indices_de_départ ne correspondent pas"

#: ../duplicity/manifest.py:473
msgid "end_index don't match"
msgstr "les indices_de_fin ne correspondent pas"

#: ../duplicity/manifest.py:478
msgid "Hashes don't match"
msgstr "Les empreintes ne correspondent pas"

#: ../duplicity/patchdir.py:73 ../duplicity/patchdir.py:80
#, python-format
msgid "Patching %s"
msgstr "En train de réparer %s"

#: ../duplicity/patchdir.py:548
#, python-format
msgid "Error '%s' patching %s"
msgstr "Erreur « %s » pendant la réparation de %s"

#: ../duplicity/patchdir.py:625
#, python-format
msgid "Writing %s of type %s"
msgstr "Écriture %s de type %s"

#: ../duplicity/path.py:109
#, python-format
msgid "Warning: %s invalid devnums (0x%X), treating as (0, 0)."
msgstr "Avertissement : %s nombres de développement non valides (0x%X), considérés comme étant (0, 0)."

#: ../duplicity/path.py:235 ../duplicity/path.py:295
#, python-format
msgid "Warning: %s has negative mtime, treating as 0."
msgstr "Attention : %s a un mtime négatif, il sera traité comme étant 0."

#: ../duplicity/path.py:359
msgid "Difference found:"
msgstr ""

#: ../duplicity/path.py:368
#, python-format
msgid "New file %s"
msgstr "Nouveau fichier %s"

#: ../duplicity/path.py:371
#, python-format
msgid "File %s is missing"
msgstr "Fichier %s manquant"

#: ../duplicity/path.py:374
#, python-format
msgid "File %%s has type %s, expected %s"
msgstr "Le fichier %%s est de type %s, %s attendu"

#: ../duplicity/path.py:379 ../duplicity/path.py:408
#, python-format
msgid "File %%s has permissions %s, expected %s"
msgstr "Le fichier %%s a les permissions %s, %s attendues"

#: ../duplicity/path.py:385
#, python-format
msgid "File %%s has mtime %s, expected %s"
msgstr "Le fichier %%s a un mtime %s, %s attendu"

#: ../duplicity/path.py:396
#, python-format
msgid "Data for file %s is different"
msgstr "les données pour le fichier %s sont différentes"

#: ../duplicity/path.py:404
#, python-format
msgid "Symlink %%s points to %s, expected %s"
msgstr "Le lien symbolique %%s pointe vers %s, %s attendu"

#: ../duplicity/path.py:411
#, python-format
msgid "Device file %%s has numbers %s, expected %s"
msgstr "Le fichier de périphérique %%s a le numéro %s, %s attendu"

#: ../duplicity/path.py:601
#, python-format
msgid "Making directory %s"
msgstr "Création du dossier %s"

#: ../duplicity/path.py:611
#, python-format
msgid "Deleting %s"
msgstr "Suppression de %s"

#: ../duplicity/path.py:620
#, python-format
msgid "Touching %s"
msgstr "Touche %s"

#: ../duplicity/path.py:628
#, python-format
msgid "Deleting tree %s"
msgstr "Suppression de l'arbre %s"

#: ../duplicity/robust.py:85
#, python-format
msgid "Error listing directory %s"
msgstr "Erreur de listage du dossier %s"

#: ../duplicity/selection.py:115
#, python-format
msgid "Skipping socket %s"
msgstr "Passer le socket %s"

#: ../duplicity/selection.py:121
#, python-format
msgid "Error initializing file %s"
msgstr "Erreur d'initialisation du fichier %s"

#: ../duplicity/selection.py:127 ../duplicity/selection.py:167
#: ../duplicity/selection.py:615
#, python-format
msgid "Error accessing possibly locked file %s"
msgstr "Erreur d'accès au fichier %s éventuellement verrouillé"

#: ../duplicity/selection.py:182
#, python-format
msgid "Warning: base %s doesn't exist, continuing"
msgstr "Attention : la base %s n'existe pas, continue"

#: ../duplicity/selection.py:184 ../duplicity/selection.py:205
#: ../duplicity/selection.py:208
#, python-format
msgid "Selecting %s"
msgstr "Sélection %s"

#: ../duplicity/selection.py:263
#, python-format
msgid "                The last file selection option is the filter option %s, which will have no\n"
"                effect as there are no subsequent file selection options. Exiting because\n"
"                this probably isn't what you meant."
msgstr "                La dernière option de sélection de fichier est l'option de filtre %s, qui n'aura pas d'effet\n"
"                car il n'y a pas d'options de sélection de fichiers suivantes. Sortie car\n"
"                ce n'est probablement pas ce que vous vouliez."

#: ../duplicity/selection.py:278
#, python-format
msgid "                Only these filter mode options were specified:\n"
"                    %s\n"
"                Case sensitive globbing is the default behaviour and so this has no effect.\n"
"                Exiting because this probably isn't what you meant."
msgstr ""

#: ../duplicity/selection.py:346
#, python-format
msgid "                Fatal Error: The file specification\n"
"                    %s\n"
"                cannot match any files in the base directory\n"
"                    %s\n"
"                Useful file specifications begin with the base directory or some\n"
"                pattern (such as '**') which matches the base directory."
msgstr "                Erreur fatale : La spécification de fichier\n"
"                    %s\n"
"                ne peut correspondre à aucun fichier dans le répertoire de base\n"
"                    %s\n"
"                Les spécifications de fichier utiles commencent par le répertoire de base ou un patron\n"
"                (comme '**') qui correspond au répertoire de base."

#: ../duplicity/selection.py:360
#, python-format
msgid "Fatal Error while processing expression\n"
"%s"
msgstr "Erreur fatale pendant le traitement de l'expression\n"
"%s"

#: ../duplicity/selection.py:378
#, python-format
msgid "Reading files to backup from list %s"
msgstr "Lecture des fichiers à sauvegarder depuis la liste %s"

#: ../duplicity/selection.py:403
#, python-format
msgid "                Files-from list contains the absolute path:\n"
"                    %s\n"
"                All paths specified in a files-from list must be given relative to the backup\n"
"                source path."
msgstr "                La liste des fichiers contient le chemin absolu :\n"
"                    %s\n"
"                Tous les chemins spécifiés dans une liste de fichiers doivent être donnés par rapport au chemin source de la sauvegarde\n"
"."

#: ../duplicity/selection.py:418
msgid "                Files-from list specified which contains no files, the backup will be empty as\n"
"                a result. Exiting as this probably isn't what you meant,"
msgstr "                Liste des fichiers spécifiés qui ne contient aucun fichier, la sauvegarde sera vide comme\n"
"                un résultat. Quitter car ce n'est probablement pas ce que vous vouliez dire,"

#: ../duplicity/selection.py:435
#, python-format
msgid "                Last selection expression:\n"
"                    %s\n"
"                only specifies that files be included.  Because the default is to\n"
"                include all files, the expression is redundant.  Exiting because this\n"
"                probably isn't what you meant."
msgstr "                Dernière expression de sélection :\n"
"                    %s\n"
"                spécifie seulement que les fichiers seront inclus. Parce que la valeur par défaut est\n"
"                inclure tous les fichiers, l'expression est redondante. Sortie parce que ce\n"
"                n'est probablement pas ce que vous vouliez."

#: ../duplicity/selection.py:497
#, python-format
msgid "Reading %s filelist %s"
msgstr "Lecture de la liste de fichiers %s %s"

#: ../duplicity/selection.py:538
#, python-format
msgid "Error compiling regular expression %s"
msgstr "Erreur en compilant l'expression régulière %s"

#: ../duplicity/tempdir.py:157
#, python-format
msgid "Using temporary directory %s"
msgstr "Utilisation du dossier temporaire %s"

#: ../duplicity/tempdir.py:201
#, python-format
msgid "Registering (mktemp) temporary file %s"
msgstr "Enregistrement (mktemp) du fichier temporaire %s"

#: ../duplicity/tempdir.py:227
#, python-format
msgid "Registering (mkstemp) temporary file %s"
msgstr "Enregistrement (mkstemp) du fichier temporaire %s"

#: ../duplicity/tempdir.py:259
#, python-format
msgid "Forgetting temporary file %s"
msgstr "Oubli du fichier temporaire %s"

#: ../duplicity/tempdir.py:262
#, python-format
msgid "Attempt to forget unknown tempfile %s - this is probably a bug."
msgstr "Tente d'oublier le fichier temporaire inconnu %s - ceci est probablement un bogue."

#: ../duplicity/util.py:176
#, python-format
msgid "Acquiring lockfile %s"
msgstr "Acquisition du fichier de verrouillage %s"

#: ../duplicity/util.py:191
#, python-format
msgid "Releasing lockfile %s"
msgstr "Libération du fichier de verrouillage %s"

